crossdock/server/server.py

```diff
@@ -77,25 +83,41 @@ def head(self):
 def make_app(server):
     return tornado.web.Application(
         [
-            (r"/", MainHandler),
-            (r"/start_trace", MainHandler, (dict(server=server, method=Server.start_trace))),
-            (r"/join_trace", MainHandler, (dict(server=server, method=Server.join_trace))),
+            (r'/', MainHandler),
+            (r'/start_trace', MainHandler, (dict(server=server, method=Server.start_trace))),
+            (r'/join_trace', MainHandler, (dict(server=server, method=Server.join_trace))),
         ], debug=True)
 
 
-# TChannel handlers
-def tchannelNotImplResponse():
-    return TraceResponse(notImplementedError="python <=> tchannel not implemented")
+def make_tchannel(port):
+    global tchannel
```

Comment:
yeah, the way it's setup right now means that the call_downstream_tchannel function needs the global tchannel function. I'll look into moving the tchannel setup and all the functions into the Server class